1.5/Source/PawnStorages/PawnStorages/Farm/Interfaces/IFarmTabParent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Verse;
+
+namespace PawnStorages.Farm.Interfaces
+{
+    public interface IFarmTabParent
+    {
+        public List<ThingDef> AllowableThing { get; }
+
+        public Dictionary<ThingDef, bool> AllowedThings { get; }
+
+        public bool Allowed(ThingDef def);
```

Comment:
Very minor, hides the def field, maybe call it potentialDef or something to differentiate.